src/files/modals/pinning-modal/PinningModal.js

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Trans, withTranslation } from 'react-i18next'
+import Button from '../../../components/button/Button'
+import Checkbox from '../../../components/checkbox/Checkbox'
+import GlyphPin from '../../../icons/GlyphPin'
+import { Modal, ModalActions, ModalBody } from '../../../components/modal/Modal'
+import { connect } from 'redux-bundler-react'
+import './PinningModal.css'
+
+export const PinningModal = ({ t, tReady, onCancel, onPinningSet, file, availablePinningServices, doSelectRemotePinsForFile, className, ...props }) => {
+  const remoteServices = useMemo(() => doSelectRemotePinsForFile(file), [doSelectRemotePinsForFile, file])
+  const [selectedServices, setSelectedServices] = useState(remoteServices)
+
+  const selectService = (key) => {
+    if (selectedServices.indexOf(key) === -1) {
+      return setSelectedServices([...selectedServices, key])
+    }
+
+    return setSelectedServices(selectedServices.filter(s => s !== key))
+  }
+  return (
+    <Modal {...props} className={className} onCancel={onCancel} >
+      <ModalBody title={t('pinningModal.title')}>
+        <div className="pinningModalContainer">
+          <button className="flex items-center pa1 hoverable-button" key={t('pinningModal.localNode')} onClick={() => selectService(t('pinningModal.localNode'))}>
+            <Checkbox className='pv3 pl3 pr1 flex-none' checked={selectedServices.includes(t('pinningModal.localNode'))}/>
+            <GlyphPin fill="teal" width={24} height={24} className="mr1 flex-shrink-0"/>
```

Comment:
```suggestion
            <GlyphPin fill="currentColor" width={32} height={32} className="mr1 aqua flex-shrink-0"/>
```
To match larger text, and aqua color from Files page row icons.

Reply:
chore: update pinning modal pin icon